index.html

```diff
@@ -65,6 +65,82 @@ <h2>Introduction</h2>
       TODO(henbos): Fill in.
     </p>
   </section>
+  <section id="rtcrtpreceiver-interface">
+    <h3>
+      <dfn>RTCRtpReceiver</dfn> extensions
+    </h3>
+    <p>
+      Let <code>RTCRtpReceiver</code> objects have a
+      <dfn>[[\PlayoutDelayHint]]</dfn> internal slot initially
+      initialized to <code>null</code>.
+    </p>
+    <pre class="idl">partial interface RTCRtpReceiver {
+  attribute double? playoutDelayHint;
+};</pre>
+    <section>
+      <h2>Attributes</h2>
+      <dl data-link-for="RTCRtpReceiver" data-dfn-for="RTCRtpReceiver"
+      class="attributes">
+        <dt><dfn><code>playoutDelayHint</code></dfn> of type <span
+          class="idlAttrType">double</span>, nullable</dt>
+        <dd>
+          <p>This attribute allows the application to give a hint to the User
+          Agent about what playout delay is acceptable. The User Agent SHOULD
+          NOT playout audio or video that is received unless this amount of time
+          has passed in seconds, allowing the User Agent to perform more or less
+          buffering than it might otherwise do. This allows to influence the
```

Comment:
The case of "empty buffer after silence" needs to be addressed - with normal jitterbuffer, the buffer will be empty after DTX silence or at the beginning of the call, and one will play out sound as soon as the first samples arrive; with delayHint, it seems that the silence->sound transition should be delayed by the value of delayHint. True or false?
If true, this is behavior different from the current UA jitter buffer behavior, so needs calling out explicitly.

Reply:
It definitely used to be true (playout audio as fast as possible after DTX mode), but it might not be true with respect to new DTX design (currently A/B testing under Field Trial). Let me double check.

Reply:
> the buffer will be empty after DTX silence or at the beginning of the call, and one will play out sound as soon as the first samples arrive

This is not necessarily true. The jitter buffer should try to maintain some delay in these cases in order to handle jitter in future packets.

Reply:
I added a "note" to explicitly call this out.